pages/fixit/about.md

```diff
@@ -0,0 +1,43 @@
+---
+permalink: /fixit/
+---
+# Fixits
+
+## What's a Fixit?
+
+18F constantly evolves as people jump in to improve our culture and products. We also use sprints to introduce new features in a short period of time. A fixit is yet another way we can collectively rally within a scheduled period of time to get things done. A fixit is slightly different than a sprint:
+
+* A fixit is more event-focused than a sprint.
+* The goal of a fixit is broader: to get something to the next plateau rather than to implement specific feature sets.
+
+## How Does a Fixit Work?
+
+1. Someone identifies a process that could benefit from a concentrated effort (for example, on-boarding) and decides to organizer a fixit.
+2. The fixit organizers compile a list of to-dos and assemble them in one place. These could be:
+    * GitHub issues
+    * Trello tasks
+    * Surveys
+2. At the appointed time, fixit participants collectively attack the to-do list, with individuals submitting a record of what they did. It's the organizers' job to make that last part easy.
```

Comment:
This is the second `2.`, but I guess they could be any number, as Markdown just does the right thing.